src/components/ConfigureFooter.vue

```diff
@@ -1,26 +1,61 @@
+<script lang="ts" setup>
+import VButton from '@/components/lib/VButton/VButton.vue'
+
+defineProps({
+  show: {
+    type: Boolean,
+    default: true,
+  },
+  saveLabel: String,
+  cancelLabel: String,
```

Comment:
```suggestion
  saveLabel: { type: String, default: 'SAVE' },
  cancelLabel: { type: String, default: 'CANCEL' }
```

Remove the default values from the component:

```diff
- :label="cancelLabel || 'CANCEL'"
+ :label="cancelLabel"
```